livedata/build.gradle

```diff
@@ -139,7 +139,7 @@ afterEvaluate {
                 // Custom attributes of the publication
                 groupId = project.group
                 artifactId = 'stream-chat-android-livedata'
-                version = '0.8.6-SNAPSHOT'
+                version = '0.8.7-SNAPSHOT'
```

Comment:
This version wouldn't need to be updated. Last released version is 0.8.5, so `0.8.6-SNAPSHOT` should be enough.
It is updated everytime that a new version is released